docs/Makefile

```diff
@@ -229,20 +240,16 @@ dummy:
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
 
+# this is only left for backwards compatibility. Changed to be consistent with windows
 .PHONY: html-api
-html-api:
+html-api: htmlapi
+
+# generate api docs for instruments automatically
+.PHONY: genapis
+genapis:
```

Comment:
shall this be documented in the `help` above as well or is it not supposed to be called "alone"?

Reply:
Exactly I don't think this is useful to be called alone but we may want to add it to other targets if we want to generate the docs in another format